applicationinsights/channel/contracts/MessageData.py

```diff
@@ -9,6 +10,19 @@ class MessageData(object):
     
     DATA_TYPE_NAME = 'MessageData'
     
+    PYTHON_LOGGING_LEVELS = {
```

Comment:
Contracts might not be the right spot for this dictionary. If we follow the other SDKs, eventually these files will all be autogenerated from our schema